kratos/tests/sources/test_condition.cpp

```diff
@@ -41,13 +41,13 @@ namespace Kratos {
             list_nodes[1] = p_node2;
             list_nodes[2] = p_node3;
 
-            auto p_cond = model_part.CreateNewCondition("Condition3D", 1, list_nodes, p_prop);
```

Comment:
well here the cleanest thing would have been to define the list_of_nodes as a PointerVector